src/lib/pubkey/dl_group/dl_group.cpp

```diff
@@ -39,6 +39,18 @@ class DL_Group_Data final {
             m_exponent_bits(dl_exponent_size(m_p_bits)),
             m_source(source) {}
 
+      DL_Group_Data(const BigInt& p, const BigInt& g, DL_Group_Source source) :
+            m_p(p),
+            m_g(g),
```

Comment:
```suggestion
      DL_Group_Data(BigInt p, BigInt g, DL_Group_Source source) :
            m_p(std::move(p)),
            m_g(std::move(g)),
```

... also in the pre-existing constructor above.